includes/discovery/sensors/state/equallogic.inc.php

```diff
@@ -141,3 +141,102 @@
         }//end if
     }//end foreach
 }//end if empty oids
+
+
+$oids_disks = snmp_walk($device, 'eqlDiskSerialNumber', '-OQn', 'EQLDISK-MIB', 'equallogic');
+
+d_echo('Disk Serials oids:\n');
+d_echo($oids_disks."\n");
+
+/*
+    .1.3.6.1.4.1.12740.3.1.1.1.5.1.329840783.1 = RZ1wTmZ00980
+    .1.3.6.1.4.1.12740.3.1.1.1.5.1.329840783.2 = RZ1wTmZ00964
+    .1.3.6.1.4.1.12740.3.1.1.1.5.1.329840783.3 = RZ1wTmZ0095E
+    .1.3.6.1.4.1.12740.3.1.1.1.5.1.329840783.4 = RZ1wTmZ0096W
+    .1.3.6.1.4.1.12740.3.1.1.1.5.1.329840783.5 = RZ1wTmZ0097U
+    .1.3.6.1.4.1.12740.3.1.1.1.5.1.329840783.6 = RZ1wTmZ008RK
+    .1.3.6.1.4.1.12740.3.1.1.1.5.1.329840783.7 = RZ1wTmZ0098D
+    .1.3.6.1.4.1.12740.3.1.1.1.5.1.329840783.8 = RZ1wTmZ0095C
+    .1.3.6.1.4.1.12740.3.1.1.1.5.1.329840783.9 = RZ1wTmZ00970
+    .1.3.6.1.4.1.12740.3.1.1.1.5.1.329840783.10 = RZ1wTmZ0094
+    .1.3.6.1.4.1.12740.3.1.1.1.5.1.329840783.11 = RZ1wTmZ85BZ
+    .1.3.6.1.4.1.12740.3.1.1.1.5.1.329840783.12 = RZ1wTmZ00958
+    .1.3.6.1.4.1.12740.3.1.1.1.5.1.329840783.13 = RZ1wTmZ00983
+    .1.3.6.1.4.1.12740.3.1.1.1.5.1.329840783.14 = RZ1wTmZ0096D
+    .1.3.6.1.4.1.12740.3.1.1.1.5.1.329840783.15 = RZ1wTmZ00952
+    .1.3.6.1.4.1.12740.3.1.1.1.5.1.329840783.16 = RZ1wTmZ00962
+    .1.3.6.1.4.1.12740.3.1.1.1.5.1.329840783.17 = RZ1wTmZ0095N
+    .1.3.6.1.4.1.12740.3.1.1.1.5.1.329840783.18 = RZ1wTmZ0097M
+    .1.3.6.1.4.1.12740.3.1.1.1.5.1.329840783.19 = 6RZ1wTmZ3QS5
+    .1.3.6.1.4.1.12740.3.1.1.1.5.1.329840783.20 = RZ1wTmZ0091H
+    .1.3.6.1.4.1.12740.3.1.1.1.5.1.329840783.21 = RZ1wTmZ00981
+    .1.3.6.1.4.1.12740.3.1.1.1.5.1.329840783.22 = RZ1wTmZ0097H
+    .1.3.6.1.4.1.12740.3.1.1.1.5.1.329840783.23 = RZ1wTmZ00959
+    .1.3.6.1.4.1.12740.3.1.1.1.5.1.329840783.24 = RZ1wTmZ0098B
+**/
+
+$base_oid         = '.1.3.6.1.4.1.12740.3.1.1.1.8.1.'; // eqlDiskStatus
+
+if (!empty($oids_disks)) {
+/*
+eqlDiskStatus
```

Comment:
👍  In that same vein, I'm also cleaning up all the $variable1 stuff and naming things appropriately.  I got a little overzealous this morning I guess and didn't bother going in and cleaning things up like I normally would have.